### PR TITLE
tests/spiffs: Move from unittests to regular test

### PR DESCRIPTION
**Pull request by bergzand**
### Contribution description

another one related to #10187

### Testing procedure

Verify that `tests/pkg_spiffs` works: `BOARD=<your-favourite> make -C tests/pkg_spiffs flash test`

### Issues/PRs references

#10187

### CODE REVIEW

File: tests/unittests/Makefile [hunk 1]
```diff
@@ -207,7 +207,7 @@ MSP430_BOARDS := chronos \
                  wsn430-v1_4 \
                  z1 \
                  #
-DISABLE_TEST_FOR_MSP430 := tests-relic tests-spiffs tests-cpp_%
+DISABLE_TEST_FOR_MSP430 := tests-relic tests-cpp_%
```
Comment (miri64) [inline tests/unittests/Makefile]: This blacklisting needs to also be moved to `tests/pkg_spiffs`.

File: tests/pkg_spiffs/tests/01-run.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import os
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    board = os.environ['BOARD']
+    # Increase timeout on "real" hardware
+    timeout = 120 if board is not 'native' else -1
+    child.expect('OK \(\d+ tests\)', timeout=timeout)
```
Comment (cladmi) [inline tests/pkg_spiffs/tests/01-run.py]: Please also directly inline changing this regex to a raw string.
```suggestion
    child.expect(r'OK \(\d+ tests\)', timeout=timeout)
```

Comment (bergzand) [reply tests/pkg_spiffs/tests/01-run.py]: Done

File: tests/unittests/Makefile (hunk 1 above)
Comment (bergzand) [reply tests/unittests/Makefile]: Should be okay now

**Comment by bergzand** [comment]
> Please squash!

Mind if I update the board blacklist/insufficient memory list while squashing?

**Comment by cladmi** [comment]
@bergzand no problem

**Comment by cladmi** [comment]
All green, let's go. No more pkgs in `tests/unittests`.

**Comment by bergzand** [comment]
Thank you!

**Pull request merged**